pkg/apis/eventing/v1/broker_defaults_test.go

```diff
@@ -531,15 +491,97 @@ func TestBrokerSetDefaults(t *testing.T) {
 				},
 			},
 		},
-		"no config, uses namespace4's broker class and config": {
+	}
+	for n, tc := range testCases1 {
+		t.Run(n, func(t *testing.T) {
+			tc.initial.SetDefaults(config.ToContext(context.Background(), defaultConfig))
+			if diff := cmp.Diff(tc.expected, tc.initial); diff != "" {
+				t.Fatal("Unexpected defaults (-want, +got):", diff)
+			}
+		})
+	}
+
+	// Add cluster-level multi-class-based configs to defaultConfig
+	brokerClasses := make(map[string]*config.BrokerConfigSpec)
+	brokerSpec1 := &config.BrokerSpec{
```

Comment:
I'd call `brokerSpec<X>` variable with the associated class for clarity like `brokerSpecMTChannelBasedBroker`.

Reply:
Got it